examples/platforms/nrf528xx/src/radio.c

```diff
@@ -107,8 +108,10 @@ static otInstance *  sInstance = NULL;
 static otRadioFrame sAckFrame;
 static bool         sAckedWithFramePending;
 
+static int8_t sMaxTxPowerTable[OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MAX - OT_RADIO_2P4GHZ_OQPSK_CHANNEL_MIN + 1];
 static int8_t sDefaultTxPower;
-static int8_t sLnaGain = 0;
+static int8_t sLnaGain          = 0;
+static bool   sPowerInitialized = false;
```

Comment:
`sDefaultTxPower` is already used when initialize this module, does it mean it's always treat initialized?

Reply:
This is used to mark whether we've set the power using `otPlatRadioSetTransmitPower`. Otherwise we will use 0dbM as the transmit power though we don't mean to.

Reply:
what if `sDefaultTxPower` is set to `OT_RADIO_POWER_INVALID`? How about initializing `sDefaultTxPower` to `OT_RADIO_POWER_INVALID` and making sure `aPower` of  otPlatRadioSetTransmitPower() is not `OT_RADIO_POWER_INVALID`?

Reply:
Done